packages/grid/test/visual/lumo/grid.test.js

```diff
@@ -6,7 +6,6 @@ import '../../../theme/lumo/vaadin-grid-selection-column.js';
 import '../../../theme/lumo/vaadin-grid-sorter.js';
 import '../../../theme/lumo/vaadin-grid-sort-column.js';
 import '../../../theme/lumo/vaadin-grid-tree-column.js';
-import '../grid.common.js';
```

Comment:
I think this is unintended, this module contains some common setup for visual tests. The same happened in `icon`. Those were the only two packages were visual tests were modified.

Reply:
Good catch, fixed.